Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+PRINT 'Begin Sql scripts execution'
+SET XACT_ABORT ON
+SET NOCOUNT ON
+
+DECLARE @_acquireLock VARCHAR(64)
+DECLARE @_acquireLockMilliseconds INT
+DECLARE @_acquireLockMaxAttempts INT
+DECLARE @_longRunningThresholdMilliSeconds INT
+DECLARE @_lastAttemptsToKillSessions INT
+DECLARE @_isHosted INT
+
+PRINT 'Adding SQL files into temporary table _tmpSqlFilesTable'
+
+IF OBJECT_ID (N'_tmpSqlFilesTable', N'U') IS NOT NULL 
+BEGIN
+   PRINT 'Temporary table exists already. Dropping table _tmpSqlFilesTable'
+   DROP TABLE _tmpSqlFilesTable
+END
+CREATE TABLE _tmpSqlFilesTable (ScriptIndex INT NOT NULL IDENTITY, SqlScriptContent NVARCHAR(MAX) NULL)
```

Comment:
>_tmpSqlFilesTable [](start = 13, length = 17)

1. These are created in customer's DB. Should it require some consent from customer?
2. Would there be any popular guidelines that say names started with _ are for certain purpose?